### PR TITLE
Fix appledoc warning

### PR DESCRIPTION
**Pull request by mdyson**
Fixing a problem with the AppleDoc in this causing a warning to appear in the main project.  I also upgraded the project file to recommended settings since they were minor.

### CODE REVIEW

**Comment by livings124** [comment]
can you revert everything but the appledoc change? don't want to set the project file to 8 until it's out

File: FSQLocationBroker/FSQLocationBroker.m [hunk 1]
```diff
@@ -26,7 +26,7 @@ @interface FSQLocationBroker ()
 @property (atomic, readwrite) NSSet *locationSubscribers;
 @property (atomic, readwrite) NSSet *regionSubscribers;
 @property (atomic, readwrite) NSSet *visitSubscribers;
-@property (nonatomic, copy) CLLocation *currentLocation;
+@property (atomic, copy) CLLocation *currentLocation;
```
Comment (livings124) [inline FSQLocationBroker/FSQLocationBroker.m]: why this change?

Comment (mdyson) [inline FSQLocationBroker/FSQLocationBroker.m]: the header had atomic and the .m file has nonatomic, it was causing a warning and I thought that was weird
